docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+volumes:
+ db-data:
+
+services:
+  mysql:
+    image: my-sql:1.2
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+     - "3306:3306"
+    volumes:
+     - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
+    networks:
+      - db-data-net
+
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
```

Comment:
The port mapping for the `pythonapp` service is consistent with the Dockerfile, which exposes port 8080. Ensure that this matches the port mentioned in the `INSTRUCTION.md` file for accessing the application.